sugar.go

```diff
@@ -340,6 +341,7 @@ func (s *SugaredLogger) sweetenFields(args []interface{}) []Field {
 	// fields, we shouldn't penalize them with extra allocations.
 	fields := make([]Field, 0, len(args))
 	var invalid invalidPairs
+	var seenError bool
```

Comment:
nit: let's combine these into a `var` block

Reply:
Sure! Thank you. Done.